src/graph/graph.c

```diff
@@ -841,7 +837,6 @@ void Graph_GetNodeEdges
 				EdgeID id = vx[i];
 
 				if(SINGLE_EDGE(id)) {
-					id = SINGLE_EDGE_ID(id);
```

Comment:
few lines below `// multi-edge entry` need to adjust `EdgeID *ids = (EdgeID*)id;` before using it